yt/geometry/geometry_handler.py

```diff
@@ -77,7 +77,7 @@ def _initialize_data_storage(self):
             writeable = os.access(dir_to_check, os.W_OK)
         else:
             writeable = os.access(fn, os.W_OK)
-        writeable = writeable and not ytcfg.getboolean("yt", "onlydeserialize")
+        writeable = writeable and not ytcfg.get("yt", "only_deserialize")
```

Comment:
hum. I asked why you changed a parameter to camel case earlier, and here you're changing to snake case. I'm not fundamentally opposed to changing the case in parameters that have more than one word, but I would like it much better that the chosen case be consistent among them.

Reply:
This is an unfortunate change indeed, I will revert it. Unfortunately, if we change the spelling of config entries, it will make the migration harder (unless we explicitly map from the old keys to the new one). If we vote for this, I would advocate in favor of using snake case everywhere.